test/disability-benefits/526EZ/config/privateMedicalRecordsRelease.unit.spec.jsx

```diff
@@ -3,12 +3,12 @@ import { expect } from 'chai';
 import sinon from 'sinon';
 import { mount } from 'enzyme';
 
-import { DefinitionTester, fillData } from '../../../util/schemaform-utils.jsx';
+import { DefinitionTester, fillData } from '../../../../src/platform/testing/unit/schemaform-utils.jsx';
 import formConfig from '../../../../src/js/disability-benefits/526EZ/config/form.js';
 import initialData from '../schema/initialData.js';
 
-describe('Disability benefits 526EZ VA facility', () => {
-  const { schema, uiSchema, arrayPath } = formConfig.chapters.supportingEvidence.pages.privateMedicalRecordsRelease;
+xdescribe('Disability benefits 526EZ VA facility', () => {
```

Comment:
Intentionally disabled?

Reply:
Yes, this file was missing the spec extension and not being run. I fixed that and the test is failing, so I explicitly ignored them. @cvalarida may want to take a look at this.

Reply:
I think @cehsu said she wrote this and was looking into it.
Thanks for keeping us in the loop! :raised_hands: